desktop/dev-environments.md

```diff
@@ -35,10 +35,14 @@ Click **Install** to download and install any missing tools.
 
 ## Start a single container Dev Environment
 
-The simplest way to get started with Dev Environments is to create a new environment by cloning the Git repository of the project you are working on. For example, let us create a new Dev Environment using a simple `singe-dev-env` project from the [Docker Samples](https://github.com/dockersamples/single-dev-env){:target="_blank" rel="noopener" class="_"} GitHub repository.
+The simplest way to get started with Dev Environments is to create a new environment by cloning the Git repository of the project you are working on. For example, let us create a new Dev Environment using a simple `single-dev-env` project from the [Docker Samples](https://github.com/dockersamples/single-dev-env){:target="_blank" rel="noopener" class="_"} GitHub repository.
+
+> **Note**
+>
+> When cloning a Git repository using SSH, ensure you've added your SSH key to the ssh-agent. To do this, open a terminal and run `ssh-add $HOME/.ssh/id_rsa`.
```

Comment:
The key isn't always `id_rsa`, maybe we should put `ssh-add <PATH>` or some other way of saying `ssh-add <path to yout public ssh key>` ?

Reply:
Updated